v2/contribute/sidebars.js

```diff
@@ -271,7 +271,10 @@ module.exports = {
             "decisions/oauth/0014",
             "decisions/oauth/0015",
```

Comment:
how would things like changes in the user's role be propagated to the client's access / id token?

Reply:
During refresh: when we are rebuilding the token payloads, the built-in claims/scopes will be refreshed as well. They are part of the payloads passed to the user callback.